x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.test.ts

```diff
@@ -564,6 +566,36 @@ describe('Response actions', () => {
         expect(responseBody.action).toBeUndefined();
       });
 
+      it('handles execute', async () => {
+        const ctx = await callRoute(
+          EXECUTE_ROUTE,
+          {
+            body: { endpoint_ids: ['XYZ'], parameters: { command: 'ls -al', timeout: 1000 } },
```

Comment:
can you also add a test with no `timeout` in the `body` and ensure that the action that is created also does not have the `timeout` property defined.